src/community/resources-and-tools/index.md.njk

```diff
@@ -11,6 +11,8 @@ You can use these community resources and tools to help you use the GOV.UK Desig
 
 The GOV.UK Design System team is not responsible for these resources and tools and we cannot support you with using them. Please contact the owner if you need help or you want to request a feature.
 
+To find information about other design systems across government, see the [list of design systems created by different departments](https://github.com/ctdesign/gov-design-systems-list). This resource is not managed by the GOV.UK Design System team.
```

Comment:
Still thinking through this a bit — particularly if there's a way to avoid repeating the 'This resource is not managed by the GOV.UK Design System team' sentence.

Perhaps by putting it in its own H2 (so it can be covered by the previous paragraph). But not sure if it gives it too much prominence.

Or perhaps rephrase and shorten the sentence a tiny bit? 🤔

Reply:
Those are good suggestions, thanks! Wonder could we just cut the second sentence ('This resource...')?

Reply:
How about:

```suggestion
To find information about other design systems across government, see a community managed [list of design systems created by different departments](https://github.com/ctdesign/gov-design-systems-list).
```

Reply:
@calvin-lau-sig7 Looks great! Thanks, will add in.